### PR TITLE
Reorganize custom container folder

### PR DESCRIPTION
**Pull request by xanwal**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Moved and renamed all custom container examples (6) 

fixes #

- Various small fixes

### CODE REVIEW

**Comment by shohei1029** [comment]
Thanks for the PR!
It seems that the subscription used in the test doesn't have enough quota or there are many deployments that are not deleted.
Do you know who can fix this? Maybe @balapv ?
image

**Review by shohei1029 — CHANGES_REQUESTED** [review]
I can approve when the build test passes. I saw several types of problems for each .sh file.

File: cli/deploy-custom-container-minimal-multimodel.sh [hunk 1]
```diff
@@ -70,10 +73,12 @@ SCORING_URL=$(az ml online-endpoint show -n $ENDPOINT_NAME --query scoring_uri -
...
-curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/test-data-diabetes.json"  $SCORING_URL
+curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/iris-test-data.json"  $SCORING_URL
+curl -H "Authorization: Bearer $KEY" -H "Content-Type: application/json" -d @"$BASE_PATH/test-data/diabetes-test-data.json"  $SCORING_URL
 # </test_online_endpoints> 
 
 # <delete_online_endpoint>
 az ml online-endpoint delete -y -n $ENDPOINT_NAME
 # </delete_online_endpoint>
+
+rm *.yml_
```
Comment (shohei1029) [inline cli/deploy-custom-container-minimal-multimodel.sh:84]: Do we need `rm *.yml_`?
```suggestion

```

Comment (xanwal) [reply cli/deploy-custom-container-minimal-multimodel.sh]: Changes have been made... runners still having problems it looks like.

**Comment by xanwal** [comment]
> I can approve when the build test passes. I saw several types of problems for each .sh file.

Alright it looks like we're all passing for this batch. Thanks for your help as always @shohei1029!!

**Review by shohei1029 — APPROVED** [review]

**Review by DannyBruno — APPROVED** [review]
If they have been tested, it looks good to me

**Pull request merged**